src/components/Testimonials/styles.ts

```diff
@@ -0,0 +1,66 @@
+import styled, { css } from 'styled-components'
+import media from 'styled-media-query'
+
+export const Wrapper = styled.section``
+export const Content = styled.section``
+
+export const Title = styled.h2`
+  ${({ theme }) => css`
+    font-size: ${theme.font.sizes.xxlarge};
+    font-weight: ${theme.font.light};
+    line-height: ${theme.font.line.xlarge};
+    margin-bottom: ${theme.spacings.xlarge};
+    text-align: center;
+
+    ${media.lessThan('medium')`
+      font-size: ${theme.font.sizes.xlarge};
+      margin-bottom: ${theme.spacings.large};
+    `}
+  `}
+`
+
+export const TestimonialsCards = styled.section`
+  ${({ theme }) => css`
+    column-gap: ${theme.grid.gutter};
+    columns: 384px 3;
+  `}
+`
+
+export const Card = styled.div`
+  ${({ theme }) => css`
+    background: ${theme.colors.bglight};
+    border-radius: ${theme.border.radius};
+    break-inside: avoid;
+    margin-bottom: ${theme.grid.gutter};
+    padding: ${theme.spacings.medium} ${theme.spacings.small};
+  `}
+`
+
+export const MessageUser = styled.p`
```

Comment:
Gostei que ao menos usou `p` que é muito melhor para texto do que `div`. Mas aqui era uma "pegadinha" para ver o conhecimento de vocês com algumas tags mais semânticas.

Para comentários, existe o q/blockquote/cite, que deveria usar aqui:
https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/